store/bolt/internal/internal.proto

```diff
@@ -0,0 +1,10 @@
+package internal;
+
+message User {
+	required uint64 ID = 1;
+	required string FirstName = 2;
+	required string LastName = 3;
+	required string Email = 4;
+}
+
+
```

Comment:
This is the on-disk format of our data. We just have `User` right now but we'll add the others soon. Although this looks like a Go file it's actually a [protocol buffers](https://developers.google.com/protocol-buffers/) definition file. We can use the `protoc` command that comes with Protocol Buffers to generate out Go code that can use this.